synapse/cmds/cortex.py

```diff
@@ -87,15 +87,37 @@ def nodevalu(t):
             return resp
 
         for node in nodes:
+
             form = node[1].get('tufo:form')
             valu = node[1].get(form)
 
-            tags = sorted(s_tufo.tags(node,leaf=True))
-            tags = [ '#'+tag for tag in tags ]
+            leafs = set(sorted(s_tufo.tags(node,leaf=True)))
```

Comment:
Can a node have more than one of the same tag? If not, we shouldn't have to unique them.